# HG changeset patch
# User Rik <octave@nomad.inbox5.com>
# Date 1319755256 25200
#      Thu Oct 27 15:40:56 2011 -0700
# Node ID 2d574194718ccc89ee0ee6f1974cba16f20e8d0c
# Parent  1d97688f852232614a14d578181b6729c2dc5fd2
namelengthmax.m: Consolidate docstring by using @math macro.  Add %!test.

* namelengthmax.m: Consolidate docstring by using @math macro.  Add %!test.

diff --git a/scripts/miscellaneous/namelengthmax.m b/scripts/miscellaneous/namelengthmax.m
--- a/scripts/miscellaneous/namelengthmax.m
+++ b/scripts/miscellaneous/namelengthmax.m
@@ -14,24 +14,21 @@
 ##
 ## You should have received a copy of the GNU General Public License
 ## along with Octave; see the file COPYING.  If not, see
 ## <http://www.gnu.org/licenses/>.
 
 ## -*- texinfo -*-
 ## @deftypefn {Function File} {} namelengthmax ()
 ## Return the @sc{matlab} compatible maximum variable name length.  Octave is
-## capable of storing strings up to
-## @tex
-## $2^{31} - 1$
-## @end tex
-## @ifnottex
-## @code{2 ^ 31 - 1}
-## @end ifnottex
-## in length.  However for @sc{matlab} compatibility all variable, function
+## capable of storing strings up to @math{2^{31} - 1} in length.
+## However for @sc{matlab} compatibility all variable, function,
 ## and structure field names should be shorter than the length supplied by
 ## @code{namelengthmax}.  In particular variables stored to a @sc{matlab} file
 ## format will have their names truncated to this length.
 ## @end deftypefn
 
 function n = namelengthmax ()
   n = 63;
 endfunction
+
+
+%!assert (namelengthmax, 63)
